front-end/src/components/Card/Card.js

```diff
@@ -1,23 +1,90 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import moment from 'moment';
+import { WatchLater, ThumbUpAlt, ThumbUpAltOutlined } from '@material-ui/icons';
+import { useDispatch, useSelector } from 'react-redux';
+import { getData } from '../../redux/nodes/authentication/selectors';
+import { likePost, unlikePost } from '../../redux/nodes/entities/posts/actions';
```

Comment:
Also re-arrange actions together and selector together

Reply:
The get data selector is actually coming from the authentication state. It has been renamed to getAuthData for better clarity.